xstate-chatbot/nodejs/src/machine/vitals.js

```diff
@@ -133,6 +134,18 @@ const vitalsFlow = {
               actions: assign((context, event) => {
                 context.slots.person.isHospitalizationRequired = 'yes';
                 context.slots.person.isSpecializationRequired = 'no';
+                context.slots.person.isPatientStable='no';
+              }),
+              target: '#doctorRemark',
+            },
+            ,
+            {
+              cond: (context) => context.intention == 'patientStable',
```

Comment:
If the target is not different, no need to write separate cond:.
You can assign different values to different slots within the same actions function. 
Write the first condition for INTENTION_UNKNOWN which will take the user to the error state.

Reply:
actually for all three options, different different three apis are called.

Reply:
Still, I think there isn't any need to make separate conditions.